src/core_read.cpp

```diff
@@ -121,6 +122,29 @@ static bool CheckTxScriptsSanity(const CMutableTransaction& tx)
     return true;
 }
 
+static bool DecodeCompressedTx(CCompressedTransaction& ctx, const std::vector<unsigned char>& ctx_data) {
+	CDataStream ssData(ctx_data, SER_NETWORK, PROTOCOL_VERSION);
+	try {
+		ssData >> ctx;
+		return true;
+	} catch (const std::exception& exc) {
+		std::cout << exc.what() << std::endl;
+ 		//TODO remove ^
+		// Fall through.
+	}
+	return false;
+}
+
+bool DecodeCompressedHexTx(CCompressedTransaction& ctx, const std::string& hex_ctx)
+{
+    if (!IsHex(hex_ctx)) {
+        return false;
+    }
+
+    std::vector<unsigned char> ctxData(ParseHex(hex_ctx));
+    return DecodeCompressedTx(ctx, ctxData);
```

Comment:
In 1d376496fe9ac7110d0fac773ae5b5d9ef32aa59:

You can do `const auto ctxData = ParseHex(hex_ctx)` or even just drop the `ctxData` variable entirely and directly `return DecodeCompressedTx(ctx, ParseHex(hex_ctx));`.

Reply:
Removed completely since this code was never used, Thanks.